src/test/java/ui/UITests.java

```diff
@@ -0,0 +1,111 @@
+package ui;
+
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.Point;
+import org.openqa.selenium.WebElement;
+import ui.helpers.ActionsHelper;
+import ui.pages.DemoApiTestsOneLaunchPage;
+import ui.pages.DemoDashboardPage;
+import ui.pages.LaunchPage;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static ui.logger.UILogger.LOGGER;
+
+
+public class UITests extends BaseUITest {
+    private LaunchPage launchPage = new LaunchPage();
+    private DemoApiTestsOneLaunchPage demoApiTestsOneLaunchPage = new DemoApiTestsOneLaunchPage();
+    private DemoDashboardPage demoDashboardPage = new DemoDashboardPage();
+
+    @Test
+    void scrollToElementWithJS() {
+        assertTrue(launchPage.openLaunchPage().scrollToDefinedElement());
+    }
+
+    @Test
+    void clickOnElementWithJS() {
+        launchPage.openLaunchPage().scrollToDefinedElement();
+        launchPage.clickOnDefinedElement();
+        assertTrue(demoApiTestsOneLaunchPage.validateDemoApiTestsOneLaunchPage());
+    }
+
+    @Test
+    void resizeWidget() {
+        LOGGER.info("Preparing test data ...");
+        widgetId1 = apiSteps.createAndAddWidget(widget);
+        widgetId2 = apiSteps.createAndAddWidget(widget);
+        LOGGER.info("Test data successfully prepared");
+
+        demoDashboardPage.openDemoDashboardPage();
+        int widgetHeight = demoDashboardPage.getWidgetSize(0).getHeight();
+        int widgetWidth = demoDashboardPage.getWidgetSize(0).getWidth();
+        demoDashboardPage.resizeWidget(0);
+        demoDashboardPage.refreshPage();
+        int widgetHeightAfterResize = demoDashboardPage.getWidgetSize(0).getHeight();
+        int widgetWidthAfterResize = demoDashboardPage.getWidgetSize(0).getWidth();
```

Comment:
A lot of magic numbers here. Give them meaningful names or move to page classes with more meaningful method names.

Reply:
renamed to getSizeOfWidgetWithIndex